getting-started.md

```diff
@@ -299,3 +299,80 @@ s3-conn   Opaque   1      7m37s
 $ aws s3 ls
 2021-04-12 19:03:32 vela-website
 ```
+
+# For GCP
+
+### Apply Provider configuration
+
+```shell
+$ export GOOGLE_CREDENTIALS=xxx;export GOOGLE_PROJECT=yyy
+
+$ sh hack/prepare-gcp-credentials.sh
+
+$ kubectl get secret -n vela-system
+NAME                                              TYPE                                  DATA   AGE
+gcp-account-creds                                 Opaque                                1      52s
+
+$ k apply -f examples/gcp/provider.yaml
+provider.terraform.core.oam.dev/default created
+
+$ kubectl apply -f examples/rbac.yaml
```

Comment:
We don't need to apply this anymore as it has been included in the chart https://github.com/oam-dev/terraform-controller/tree/master/chart/templates.

Reply:
Ok. I removed the RBAC appy from the AWS and GCP example.